ui/src/shared/actions/schemaThunks.ts

```diff
@@ -3,14 +3,15 @@ import {Dispatch} from 'react'
 // import {fromFlux as parse} from '@influxdata/giraffe'
 
 // API
-// import {runQuery} from 'src/shared/apis/query'
+import {runQuery} from 'src/shared/apis/query'
```

Comment:
this isn't being used

Reply:
as in we shouldn't be using this to run the query? It's used here:

https://github.com/influxdata/influxdb/pull/19450/files/0e33e60063b1fab75c9186be801d5a26cf358532#diff-7003025bc3963a7870d490547eb9c4c0R47

Reply:
nm then i got confused